src/main/resources/openapi/NotificationsAPIs.yaml

```diff
@@ -1304,6 +1402,46 @@ components:
           example: Please provide a valid document within 28 days
       required:
         - documents
+    RejectedOverseasDocumentPersonalisation:
+      title: RejectedOverseasDocumentPersonalisation
+      type: object
+      description: Personalisation details for rejected overseas document template
+      allOf:
+        - $ref: '#/components/schemas/BasePersonalisation'
+      properties:
+        documents:
+          type: array
+          items:
+            $ref: '#/components/schemas/RejectedDocument'
+          minItems: 1
+        rejectedDocumentFreeText:
+          type: string
+          description: Free text message to provided further information regarding the document rejection
+          example: Please provide a valid document within 28 days
+      required:
+        - documents
+    BaseOverseasPersonalisation:
```

Comment:
Is this any different from BasePersonalisation?

Reply:
No I will remove